### PR TITLE
Updates footer link

### PR DESCRIPTION
**Pull request by OpenGlobe**
Direct URL link to DOI privacy policy.

### CODE REVIEW

File: ekip/everykid/templates/base.html [hunk 1]
```diff
@@ -51,8 +51,17 @@
             <a href="{% url 'about_ekip' %}">About Every Kid in a Park</a>
           </li>
           <li>
-            <a href="{% url 'privacy_policy' %}">Privacy policy</a>
+            <a href="https://www.doi.gov/privacy">Privacy policy</a>
```
Comment (khandelwal) [inline ekip/everykid/templates/base.html:54]: Should this go to our (simpler) privacy policy page? That has some information about kids and educators and then links out to the DOI policy? Does that make sense?

**Comment by OpenGlobe** [comment]
Told that it had to link directly to the DOI site, and that these others had to be included.  I asked about just including on the existing page too.

**Comment by khandelwal** [comment]
That doesn't make sense. We have information in addition to the privacy policy.

**Comment by OpenGlobe** [comment]
It wasn't presented as optional.

**Pull request merged**